docs/readthedocs/source/doc/LLM/Quickstart/benchmark_quickstart.md

```diff
@@ -0,0 +1,66 @@
+# Benchmarking
+
+We can do benchmarking on Intel GPUs and CPUs. You can refer to [this](https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install.html) to prepare the environment.
```

Comment:
We can do benchmarking on Intel CPUs and GPUs using the benchmark script we provide.

Reply:
prepare the environment for bigdl-llm.